egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000
```

Comment:
If this code depends on external matlab codes.. check for their existence and die if not found.
Give the user an option to specify the location of the required files.